xdebug 3.2.0 requires linux-headers (rtnetlink.h)

### Version of install-php-extensions

1.5.35

### Error description

```
### INSTALLING REMOTE MODULE xdebug ###
  - Installing xdebug (latest-stable): Downloading (100%)
+-----------------------------------+-------------+
| Package name                      | xdebug      |
| Package version (current release) | 3.2.0alpha2 |
| Package status                    | alpha       |
+-----------------------------------+-------------+
The following error(s) happened: configure failed, see log at /tmp/pickle.tmp/pickle-5a3034e49661a53d918922d82cd16737c598540a\config.log
1: phpize
2: Configuring for:
2: PHP Api Version:         20200930
2: Zend Module Api No:      20200930
2: Zend Extension Api No:   420200930
2: configure.ac:18: warning: $as_echo is obsolete; use AS_ECHO(["message"]) instead
2: build/php.m4:2072: PHP_CONFIG_NICE is expanded from...
2: configure.ac:18: the top level
2: configure.ac:161: warning: The macro `AC_LANG_C' is obsolete.
2: configure.ac:161: You should run autoupdate.
2: ./lib/autoconf/c.m4:72: AC_LANG_C is expanded from...
2: build/libtool.m4:2728: _LT_AC_LANG_C_CONFIG is expanded from...
2: build/libtool.m4:2727: AC_LIBTOOL_LANG_C_CONFIG is expanded from...
[...]
2: checking for poll.h... yes
2: checking for sys/poll.h... yes
2: checking for linux/rtnetlink.h... no
2: configure: error: rtnetlink.h is required, install the linux-headers package: apk add --update linux-headers
```

### Docker image

php:8.0-cli-alpine

### Minimal Dockerfile

```Dockerfile
FROM php:8.0-cli-alpine

ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
RUN chmod +x /usr/local/bin/install-php-extensions

RUN install-php-extensions xdebug
```

--- a/install-php-extensions
+++ b/install-php-extensions
@@ -1292,6 +1292,11 @@ buildRequiredPackageLists() {
 			wddx@debian)
 				buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile libxml2-dev"
 				;;
+			xdebug@alpine)
+				if test $PHP_MAJMIN_VERSION -ge 800; then
+					buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile linux-headers"
+				fi
+				;;
 			xlswriter@alpine)
 				buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile zlib-dev"
 				;;

